.github/workflow/php.yml

```diff
@@ -0,0 +1,19 @@
+name: CI
+on: [pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      max-parallel: 3
+      matrix:
+        php-versions: ['5.6', '7.3']
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+    - name: PHP syntax checker 7.3
+      run: find . -type f -name '*.php' ! -path "./vendor/*" ! -path "./tests/*" -exec php -l -n {} \; | (! grep -v "No syntax errors detected")
```

Comment:
just FYI we probably gonna use `https://github.com/PrestaShop/github-action-php-lint` but right now it doesnt seems to properly work so let's keep this for now